doc/contracts.rst

```diff
@@ -0,0 +1,105 @@
+NuCypher contracts
+========================
+* `NuCypherToken` contract is the ERC20 token contract with additional function - burn own tokens (only for owners)
+* `MinersEscrow` contract holds Ursula's stake, stores information about Ursulas activity and assigns a reward for participating in NuCypher network. The `Issuer` contract is part of the `MinersEscrow` and uses only to split code
+* `PolicyManager` contract holds policies fee and distributes fee by periods
+* `Upgradeable` is base contract for upgrading (<nucypher.blockchain.eth/project/contracts/proxy/README.MD>)
+* `Dispatcher` contract is used as proxy to other contracts. This provides upgrading of the `MinersEscrow`, `PolicyManager` and `Government` contracts
+* `Government` contract holds ownership for dispatchers and provides voting for upgrading of those contracts
+* `UserEscrow` contract locks tokens for some time. In that period tokens are lineraly unlocked and all tokens can be used as a stake in the `MinersEscrow` contract
```

Comment:
The `UserEscrow` should be changed from linear unlocking to unlocking all in the specified time. It should also simplify the contract

Reply:
I changed the unlock. initialDeposit uses duration as input parameter, is it ok? Or should be end time?

Reply:
What's the difference (what's the "end time"?)

Reply:
I mean, endTime = blockTime + duration or endTime comes from parameter